com.unity.render-pipelines.high-definition/Documentation~/HDRP-Features.md

```diff
@@ -311,6 +385,45 @@ The Render Pipeline Debugger contains many debugging and visualization tools to
 ![](Images/HDRPFeatures-LookDev.png)
 The LookDev is a viewer that allows you to import and display Assets in a good, consistent lighting environment. Use it to validate outsourced Assets or to showcase your own created Asset with HDRP. For more information on the LookDev, including a description of how to use it, see the [LookDev documentation](Look-Dev.md).
 
+### Debug modes
+
+HDRP includes debug modes that can help you to set the correct exposure for your Scene.
```

Comment:
=> HDRP includes debug modes that can help you to debug your assets, your lighting, your scene.